typescript/src/messages/callback_types.ts

```diff
@@ -25,8 +25,8 @@ export type OnIntermediateCatchEventReachedCallback =
 export type OnIntermediateCatchEventFinishedCallback =
   (intermediateCatchEventFinished: IntermediateCatchEventFinishedMessage) => void | Promise<void>;
 
-export type OnCallActivityWaitingCallback = (callActivityWaiting: CallActivityReachedMessage) => void | Promise<void>;
-export type OnCallActivityFinishedCallback = (callActivityFinished: CallActivityFinishedMessage) => void | Promise<void>;
+export type OnActivityReachedCallback = (callActivityWaiting: ActivityReachedMessage) => void | Promise<void>;
```

Comment:
```suggestion
export type OnActivityReachedCallback = (callActivityReached: ActivityReachedMessage) => void | Promise<void>;
```

So gemeint, oder habe ich es falsch verstanden?

Reply:
Nope, genau richtig. 👍